rfc/5/index.md

```diff
@@ -9,495 +9,667 @@ responses/index
-contrast, discrete axes (`"discrete" : true`) may be indexed only by integers. Axes of representing a `channel`, `coordinate`, or `displacement` are
-usually discrete.
+Arrays are inherently discrete (see Array coordinate systems, below)
+but are often used to store discrete samples of a continuous variable.
+The continuous values "in between" discrete samples can be retrieved using an *interpolation* method.
+If an axis is continuous (`"discrete" : false`), it indicates that interpolation is well-defined.
+Axes representing `space` and `time` are usually continuous.
+Similarly, joint interpolation across axes is well-defined only for axes of the same `type`.
+In contrast, discrete axes (`"discrete" : true`) may be indexed only by integers.
+Axes representing a channel, coordinate, or displacement are usually discrete.
 
-Note: The most common methods for interpolation are "nearest neighbor", "linear", "cubic", and "windowed sinc". Here, we refer
-to any method that obtains values at real-valued coordinates using discrete samples as an "interpolator". As such, label images
-may be interpolated using "nearest neighbor" to obtain labels at points along the continuum.
+```{note}
+The most common methods for interpolation are "nearest neighbor", "linear", "cubic", and "windowed sinc".
+Here, we refer to any method that obtains values at real-valued coordinates using discrete samples as an "interpolator".
+As such, label images may be interpolated using "nearest neighbor" to obtain labels at points along the continuum.
+```
 
+#### Array coordinate systems
 
-### Array coordinate systems
+The dimensions of an array do not have an interpretation
+until they are associated with a coordinate system via a coordinate transformation.
+Nevertheless, it can be useful to refer to the "raw" coordinates of the array.
+Some applications might prefer to define points or regions-of-interest in "pixel coordinates" rather than "physical coordinates," for example.
+Indicating that choice explicitly will be important for interoperability.
+This is possible by using **array coordinate systems**.
 
-Every array has a default coordinate system whose parameters need not be explicitly defined.  Its name is the path to the array
-in the container, its axes have `"type":"array"`, are unitless, and have default "name"s. The ith axis has `"name":"dim_i"`
-(these are the same default names used by [xarray](https://docs.xarray.dev/en/stable/user-guide/terminology.html)).
+Every array has a default coordinate system whose parameters need not be explicitly defined.
+The dimensionality of each array coordinate system equals the dimensionality of its corresponding Zarr array.
+Its name is the path to the array in the container,
+its axes have `"type": "array"`, are unitless, and have default names.
+The i-th axis has `"name": "dim_i"` (these are the same default names used by [xarray](https://docs.xarray.dev/en/stable/user-guide/terminology.html)).
```

Comment:
```suggestion
The i-th axis has `"name": "dim_i"`.
```
I don't think it's relevant what xarray's defaults are, and also xarray might change their defaults, which would bring this section out of date. Unless xarray compatibility is an explicit design goal we should remove references to xarray from this section (fine to talk about xarray outside the normative section of the spec).